k4MarlinWrapper/scripts/convertMarlinSteeringToGaudi.py

```diff
@@ -254,15 +247,22 @@ def generateGaudiSteering(tree):
   return lines
 
 
+def escapeIllegalChars(file_str):
+  return file_str.replace("&", "&amp;")
```

Comment:
```suggestion
  return file_str.replace("&&", "&amp;&amp;")
```
Maybe we should consider only escaping double ampersands, as single ampersands could still be valid use-cases of xml escaping(?). At least #35 only tripped over the double ampersands, so it might be enough.

Reply:
Indeed, if some valid escaping goes into the XML like `&amp;` or `&quot;`, that is going to mess things up. So if we can just cover the case with double ampersands it will be much simpler.